actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -107,31 +106,24 @@ def _process_options(options)
       options
     end
 
-    # Process the rendered format.
-    # :api: private
-    def _process_format(format)
```

Comment:
This method is here to define a common API between subclasses of AbstractController, we should not remove.